include/benchmark/benchmark.h

```diff
@@ -655,6 +655,17 @@ enum AggregationReportMode
       ARM_FileReportAggregatesOnly | ARM_DisplayReportAggregatesOnly
 };
 
+enum Skipped
+#if defined(BENCHMARK_HAS_CXX11)
+    : unsigned
+#else
```

Comment:
nit: we can drop the #else

Reply:
sure, i just copied the pattern from the previous enum.